orbstation/code/cleanup/recipes.dm

```diff
@@ -1,27 +1,27 @@
 // Hides some things we don't want to be craftable
 /datum/crafting_recipe/tailclub
-	always_available = FALSE
+	non_craftable = FALSE
```

Comment:
I see this property name changed, should the value be inverted?
The name seems to imply the opposite of what it did before, but I am not sure.

Reply:
You are correct! I will fix this soon.